core/adapters/sqldatawarehouse.ts

```diff
@@ -122,4 +122,12 @@ export class SQLDataWarehouseAdapter extends Adapter implements IAdapter {
      ) 
      as ${table.query}`;
   }
+
+  public insertInto(target: dataform.ITarget, columns: string[], query: string) {
```

Comment:
Can you add a validation rule for sql data warehouse that throws an error during compilation if uniqueKeys is specified?

Reply:
Have added to `core/utils`